packages/page-council/src/Motions/Voting.tsx

```diff
@@ -61,9 +57,15 @@ function Voting ({ hash, idNumber, isDisabled, members, prime, proposal, votes }
     : api.tx.council.close(hash, idNumber);
 
   // vote and close if this vote ends the vote
-  const extrinsic = willPass || willFail
-    ? api.tx.utility.batch([voteExtrinsic, closeExtrinsic])
-    : voteExtrinsic;
+  const extrinsic = useMemo(() =>
```

Comment:
This is great, we still have the vote stuff above doing the same - so this will execute each render anyway since those objects will change. You probably want to encapsulate the close/vote extrinsics inside this memo. (Not needed externally)

Reply:
Thanks, totally makes sense.